### ISSUE TITLE
Minor bugs/suggestions on Plotting page

### ISSUE THREAD

**Issue by thouska** [jlu-ilr-hydro/odmf#102]
Bugs:
- Setting "ylim" has no effect on plot
- Setting of "Subplot columns" under preferences has no effect
- Field YYYY in "start" and "end" under properties takes six digits

Optional suggestions:
- Rename "file" to "Load/Save Plot" and include option to create new folders
- Default setting as "solid line" for line style instead of "None"
- Sort value types alphabeticaly
- Default setting under properties end" for today and "start" as one year ago
- Remove "properties" field and show all options in left panel instead

**Comment by philippkraft** [comment]
Bugs:
 - ylim - fixed in development
 - works for me (evtl. only after fixing ylim)
 - known problem/feature of html5 : see https://stackoverflow.com/questions/24603919/html5-date-input-6-digit-year

Optional suggestions
- File->Load /Save -> fixed
- Default solid line -> fixed
- Sort value types alphabetically: not yet fixed, but alphabetically sorted valuetypes would be great every where, new issue.
- fixed in plot.js:gettime()
- Won't fix: content of side panel gets to large. Moving only start / end time to side panel: PR welcome
